src/Public/Routes.ps1

```diff
@@ -1433,12 +1453,21 @@ function Add-PodeStaticRouteGroup {
         $DownloadOnly,
 
         [switch]
-        $RedirectToDefault
+        $RedirectToDefault,
+
+        [Parameter(Mandatory = $true, ParameterSetName = 'File')]
+        [string]
+        $FilePath
```

Comment:
This `-FilePath` parameter is missing the `.PARAMETER` definition above the function